src/main/java/teammates/ui/constants/LogType.java

```diff
@@ -0,0 +1,39 @@
+package teammates.ui.constants;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import teammates.common.util.Const;
+
+/**
+ * Constant values for all the different log types.
+ */
+public enum LogType {
```

Comment:
`LogType` is used not only as output for retrieval of logs, but also in the creation of logs, which is why it was decided that it be part of `teammates.ui.constants` to serve both purposes -- subclassing it would confine it to `api-output` only, which might not be so good for log creation requests